hardware/ASyncTCP.cpp

```diff
@@ -4,50 +4,56 @@
 #include <boost/asio.hpp>
 #include <boost/bind.hpp>
 #include <boost/system/error_code.hpp>     // for error_code
+#include "../main/Helper.h"                // for SetThreadName
```

Comment:
Please undo this, and also the log option as discussed

Reply:
"mainworker" is not a good threadname which helps when debugging, so please suggest how to set the threadname without this dependency?